app/src/main/java/swati4star/createpdf/util/FileUtils.java

```diff
@@ -0,0 +1,81 @@
+package swati4star.createpdf.util;
+
+import android.content.Context;
+import android.os.Environment;
+import android.util.Log;
+
+import com.bumptech.glide.load.engine.Resource;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+import swati4star.createpdf.R;
+
+
+
+public class FileUtils {
+
+    Context context;
+
+    public FileUtils(Context context){
+        this.context=context;
+    }
+
+    public void sortByNameAlphabetical(ArrayList<File> pdfsFromFolder) {
```

Comment:
Add java doc comments to each function in this utils file.

For eg, for this particular function

```
/**
* Sorts the given file list in increasing alphabetical  order
*
* @param pdfsFromFolder list of files to be sorted
*/

```